### PR TITLE
created branch for design logged in tabbar and pushed a test change

### PR DESCRIPTION
**Pull request by kd798529**
### What does this PR do?
Added UITabBarController with dummy viewcontrollers to the app. The view controllers have different background colors.  I have attached screenshots below

### Screenshots
image
image
image

image
image
image

### CODE REVIEW

**Review by stepharski — APPROVED** [review]

File: Chirp/ViewController.swift [hunk 1]
```diff
@@ -7,13 +7,60 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UITabBarController {
```
Comment (EMacco) [inline Chirp/ViewController.swift:10]: Could we please create a new file and class instead? You can call it `HomeTabBarController`

File: Chirp/ViewController.swift [hunk 2]
```diff
@@ -7,13 +7,60 @@
...
+        let vc1 = FirstViewVC()
+        let vc2 = SecondViewVC()
+        let vc3 = ThirdViewVC()
+        self.setViewControllers([vc1, vc2, vc3], animated: true)
+        
+        guard let items = self.tabBar.items else {
+            return
+        }
+        
+        let tabIconImages = ["ellipsis.message.fill", "person.2.fill", "squareshape.split.2x2"]
```
Comment (EMacco) [inline Chirp/ViewController.swift:28]: Can we create an extension of UIImage and add create the icons there instead?
If you pull the latest version of development and rebase, you'll notice that there's an `Extensions` folder. Inside this folder, create a new file `UIImage+Extension` and add these icons there.
E.g
```swift
extension UIImage {
    static let ellipsisMessageFill = UIImage(systemName: "ellipsis.message.fill")!
    static let personFill = UIImage(systemName: "person.2.fill")!
    ...
}
```

File: Chirp/ViewController.swift [hunk 3]
```diff
@@ -7,13 +7,60 @@
...
+        
+        for i in 0..<items.count {
+            items[i].image = UIImage(systemName: tabIconImages[i])
```
Comment (EMacco) [inline Chirp/ViewController.swift:31]: Can we set filled images for selected state and outlined for the regular state?
```suggestion
            items[i].image = UIImage(systemName: tabIconImages[i])
            items[i].selectedImage = UIImage(systemName: tabIconSelectedImages[i])
```

File: Chirp/ViewController.swift [hunk 4]
```diff
@@ -7,13 +7,60 @@
...
+        }
```
Comment (EMacco) [inline Chirp/ViewController.swift:32]: Can we also set the tintColor and unSelected color to `.appBrown` and `.appBrownSecondary` You'll have access to both of these when you rebase with the latest development 
```swift
tabBar.tintColor = .brown
tabBar.unselectedItemTintColor = .lightGray
```

File: Chirp/ViewController.swift [hunk 5]
```diff
@@ -7,13 +7,60 @@
...
 }
 
+
+class ThirdViewVC: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .brown
+    }
+}
```
Comment (EMacco) [inline Chirp/ViewController.swift:66]: Let's remember to remove these when the real classes are created For now we can leave it here

**Review by EMacco — CHANGES_REQUESTED** [review]

**Review by EMacco — APPROVED** [review]

**Pull request merged**